BasicPlaybackSampleApp/BasicPlaybackSampleApp/Players/BasicSimplePlayerViewController.m

```diff
@@ -81,6 +85,19 @@ - (void) notificationHandler:(NSNotification*) notification {
         [notification name],
         [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
         [self.ooyalaPlayerViewController.player playheadTime], appDel.count);
+  
+  //In QA Mode , adding notifications to the TextView
+  if(self.qaModeEnabled == YES) {
+  NSString *message = [NSString stringWithFormat:@"Notification Received: %@. state: %@. playhead: %f count: %d",
```

Comment:
could you reuse the message for the NSLog above too? Like
```
 NSString *message = [NSString stringWithFormat:@"Notification Received: %@. state: %@. playhead: %f count: %d", ...];
NSLog(message);
...
```
This way you won't build two NSString instances for each notification ;)